src/wasm/wasm.cpp

```diff
@@ -1294,6 +1294,14 @@ Index Function::getLocalIndex(Name name) {
   return iter->second;
 }
 
+Index* Function::getLocalIndexOrNull(Name name) {
```

Comment:
This feels a bit awkward, but matches the other get*OrNull methods. Alternatively, we could do hasLocalIndex(Name name). Any preferences?

Reply:
Function already should have `hasLocalName`

Reply:
I think the `hasLocalIndex(Name)` option would be more consistent with the methods that are already there. The `get*OrNull` make more sense when the `*` is a pointer anyway. If I were writing this today, I would probably use `std::optional` instead of separate `has*` and `get*` methods, but changing that probably isn't worth the effort.